eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,98 @@
+$Language = "go"
+$packagePattern = "go.mod"
+$RELEASE_TITLE_REGEX = "(?<releaseNoteTitle>^\#+\s+(?<version>v$([AzureEngSemanticVersion]::SEMVER_REGEX))(\s+(?<releaseStatus>\(.+\))))"
+$AUTOREST_VERSION_REGEX = "^module-version:\s+(?<version>$([AzureEngSemanticVersion]::SEMVER_REGEX))"
+
+function Get-go-PackageInfoFromPackageFile ($pkg, $workingDirectory)
+{
+    $workFolder = $($pkg.Directory)
+    $releaseNotes = ""
+    $namespaceName = $workFolder.Name
+    $rpName = $workFolder.Parent.Name
+
+    $pkgId = "sdk/$rpName/$namespaceName"
+    $pkgVersion = Get-Version($workFolder)
+
+    $changeLogLoc = @(Get-ChildItem -Path $workFolder -Recurse -Include "CHANGELOG.md")[0]
+    if ($changeLogLoc)
+    {
+        $releaseNotes = Get-ChangeLogEntryAsString -ChangeLogLocation $changeLogLoc -VersionString v$pkgVersion
+    }
+
+    $resultObj = New-Object PSObject -Property @{
+        PackageId      = $pkgId
+        PackageVersion = $pkgVersion
+        ReleaseTag     = "$($pkgId)/v$($pkgVersion)"
+        Deployable     = $true
+        ReleaseNotes   = $releaseNotes
+    }
+
+    return $resultObj
+}
+
+# get version from go config auterest.md
+function Get-Version ($pkgPath)
+{
+    $content = Get-Content(Join-Path $pkgPath "autorest.md")
```

Comment:
It ook s like this only applies to a handfull of management packages. Shoul there also be logic for retrieving version from version.go files?

Reply:
I have changed the Get-Version method to support both data plane and mgmt plane. For mgmt plane of go track2, version can be obtained by autorest.md. For data plane, version can be obtained by version.go file. Maybe two version aquisition method can be unified if code gen add version.go to mgmt plane in the future. @jhendrixMSFT @ArcturusZhang